docs/tutorials/control/air-quality-fleet.md

```diff
@@ -20,766 +20,718 @@ cost: 200
+let apiKeyId = "";
+let apiKeySecret = "";
+let hostname = "";
+let machineId = "";
 
-   {{< /expand >}}
+async function main() {
+  const opts: VIAM.ViamClientOptions = {
+    serviceHost: "https://app.viam.com",
+    credentials: {
+      type: "api-key",
+      payload: apiKeySecret,
+      authEntity: apiKeyId,
+    },
+  };
 
-## Test data sync
+  // <Insert data client and query code here in later steps>
 
-Next, check that data is being synced from your sensors to the cloud:
+  // <Insert HTML block code here in later steps>
+}
 
-1.  Open your [**DATA** page](https://app.viam.com/data).
-2.  Click the **Sensors** tab within the data page.
-3.  If you have sensor data coming from machines unrelated to this project, use the filters on the left side of the page to view data from only your air quality sensors.
-    Click the **Tags** dropdown and select the `air-quality` tag you applied to your data.
-    You can also use these filters to show the data from one of your air quality sensors at a time by typing a machine name into the **Machine name** box and clicking **Apply** in the lower-left corner.
+// <Insert getLastFewAv function definition here in later steps>
 
-    ![The sensor readings that have synced to the DATA page.](/tutorials/air-quality-fleet/synced-data.png)
+document.addEventListener("DOMContentLoaded", async () => {
+  machineId = window.location.pathname.split("/")[2];
+  ({
+    id: apiKeyId,
+    key: apiKeySecret,
+    hostname: hostname,
+  } = JSON.parse(Cookies.get(machineId)!));
 
-Once you've confirmed that data is being collected and synced correctly, you're ready to start building a dashboard to display the data.
-If you'd like to graph your data using a Grafana dashboard, try our [Visualize Data with Grafana tutorial](/tutorials/services/visualize-data-grafana/).
-If you'd like to create your own customizable dashboard using the Viam TypeScript, continue with this tutorial.
+  main().catch((error) => {
+    console.error("encountered an error:", error);
+  });
+});
+```
 
-## Code your custom TypeScript dashboard
+{{% /tablestep %}}
+{{% tablestep number=2 %}}
 
-The [Viam TypeScript SDK](https://ts.viam.dev/) allows you to build custom web interfaces to interact with your machines.
-For this project, you'll use it to build a page that displays air quality sensor data for a given location.
-You'll host the website locally on your personal computer, and view the interface in a web browser on that computer.
+For developing your app on localhost, **add the same information to your browsers local storage**.
 
-As you'll find out in the [authentication step](#authenticate-your-code-to-your-viam-app-location), you can set each customer up with credentials to access the data from only their location, or you can create a dashboard showing data from all sensors in your entire organization.
+Navigate to [Camera Viewer](https://air-quality_naomi.viamapplications.com/) and log in, then select your development machine.
```

Comment:
Same comment as other PR: It's confusing to tell them to seemingly go look at cameras without explanation of what this link is for. But sounds like it's about to change so can disregard so long as it'll get fixed.